doc/panes/fileexplorer.rst

```diff
@@ -81,7 +81,7 @@ File associations
 =================
 
 :guilabel:`Files` allows you to associate different external applications with specific file extensions they can open.
-Under the :guilabel:`File associations` tab of the :guilabel:`Files` preferences pane, you can add file types and set the external program used to open each of them by default.
+Under the :guilabel:`Preferences --> File associations` tab of the :guilabel:`Files` preferences pane, you can add file types and set the external program used to open each of them by default.
```

Comment:
I'm unclear on the motivation for this change was made, since it implies two conflicting potential interpretations, neither of which is correct (unlike the previous text):

- As mentioned, the existing text already states that this is the `File associations` tab of the `Files` preferences pane, so its confusing why "Preferences" is repeated again here
- On first reading, this change implies that `File associations` is a pane underneath preferences, but that's not the case as mentioned
- Furthermore, in context as written, it alternatively implies there is a `Preferences` in the `Files` preferences pane (and underneath that a `File associations` tab), but that is of course not correct either
- Additionally, `:guilabel:` is for single, specific GUI labels; to support multiple levels of things you'd need `:menuselection:`

Without more context on the desired intent here, I can propose either a revert to the existing, correct text which explicitly specifies that this is a tab of a specific pane under preferences:

```suggestion
Under the :guilabel:`File associations` tab of the :guilabel:`Files` preferences pane, you can add file types and set the external program used to open each of them by default.
```

Or, we _could+ specify the complete path all in one go, which is more direct and concise but lacks explicit cues for the reader as to what they should be looking for at each level, and also runs into the issue you raised in #365 which will take some engineering work to solve, and is de-facto gated by the implementation of the new theme, so I don't recommend this for now:

```suggestion
Under :menuselection:`Preferences --> Files --> File associations`, you can add file types and set the external program used to open each of them by default.
```

If you could help me understand better the problem this is intended to solve, I could try to help propose a better solution. Thanks!